src/game_map.cpp

```diff
@@ -193,7 +193,7 @@ void Game_Map::Setup(int _id) {
 			break;
 		}
 		if (parent_index < 0) {
-			Output::Warning("Map %d has invalid parent id %d!", Data::treemap.maps[current_index].parent_map);
+			Output::Warning("Map %d has invalid parent id %d!", current_index, Data::treemap.maps[current_index].parent_map);
```

Comment:
@fmatthew5876 
I'm not 100% sure if the variable I added here is the correct one.

Reply:
You want the id of the map, not the index since maps can have gaps.

`Data::treemap.maps[current_index].ID`

Reply:
thx, changed it